uber/model_checks.py

```diff
@@ -57,8 +57,25 @@ def dealer_description(group):
 
 @prereg_validation.Group
 def dealer_address(group):
-    if group.tables and not (group.address1 and group.city and group.region and group.country and group.zip_code):
```

Comment:
More fine grained address validation. Derived largely from the attendee address validation found lower down in this file.